src/Windowing/Silk.NET.Windowing.Desktop/GlfwMonitor.cs

```diff
@@ -66,5 +67,19 @@ public float Gamma
                 GlfwProvider.GLFW.Value.SetGamma(Handle, value);
             }
         }
+
+        public VideoMode[] GetAllVideoModes()
+        {
+            var rawVideoModes = GlfwProvider.GLFW.Value.GetVideoModes(Handle, out var count);
+

```

Comment:
I wonder if we can ignore the count parameter at all. As C# arrays have a length the count parameter makes only sense in C imho. By discarding the count parameter with `out _` we also can get rid of the for loop below and transform it to a one-liner with Linq's `Select`. So the whole method body can be transformed into a short one-liner.

Reply:
But we need a null check here if I understand the glfw docs correctly.

Reply:
rawVideoModes isn't a C# array, but a pointer. We need Count because it lacks the length info needed for a foreach/linq expression.

Reply:
Ok didn't check it as I'm at mobile phone atm. Then it's OK.